contao/dca/tl_module.php

```diff
@@ -11,5 +11,13 @@
  */
 
 // Add palettes to tl_module
-$GLOBALS['TL_DCA']['tl_module']['palettes']['logo']            = '{title_legend},name,headline,type;{logo_legend},imgSize;{template_legend:hide},customTpl;{protected_legend:hide},protected;{expert_legend:hide},guests,cssID';
+$GLOBALS['TL_DCA']['tl_module']['palettes']['logo']            = '{title_legend},name,headline,type;{logo_legend},imgSize,companyLogoAddLink;{template_legend:hide},customTpl;{protected_legend:hide},protected;{expert_legend:hide},guests,cssID';
 $GLOBALS['TL_DCA']['tl_module']['palettes']['socialmedialist'] = '{title_legend},name,headline,type;{template_legend:hide},customTpl;{protected_legend:hide},protected;{expert_legend:hide},guests,cssID';
+
+// Add fields to tl_module
+$GLOBALS['TL_DCA']['tl_module']['fields']['companyLogoAddLink'] = [
```

Comment:
```suggestion
$GLOBALS['TL_DCA']['tl_module']['palettes']['logo']            = '{title_legend},name,headline,type;{logo_legend},imgSize,companyLogoRedirect;{template_legend:hide},customTpl;{protected_legend:hide},protected;{expert_legend:hide},guests,cssID';
$GLOBALS['TL_DCA']['tl_module']['palettes']['socialmedialist'] = '{title_legend},name,headline,type;{template_legend:hide},customTpl;{protected_legend:hide},protected;{expert_legend:hide},guests,cssID';

// Add fields to tl_module
$GLOBALS['TL_DCA']['tl_module']['fields']['companyLogoRedirect'] = [
```
I think this name describes it better